src/fidesops/service/connectors/sql_connector.py

```diff
@@ -187,6 +196,58 @@ def create_client(self) -> Engine:
             echo=not self.hide_parameters,
         )
 
+    def set_schema(self, connection: Connection) -> None:
+        """Sets the search_path for the duration of the session"""
+        config = RedshiftSchema(**self.configuration.secrets or {})
+        if config.db_schema:
+            logger.info("Setting Redshift search_path before retrieving data")
+            connection.execute("SET search_path TO %s" % config.db_schema)
```

Comment:
We should not send raw strings to sql - should use a sql alchemy textClause here

Reply:
good catch thank you